packages/flutter_tools/test/base/logger_test.dart

```diff
@@ -131,21 +131,18 @@ void main() {
         Stdio: () => mockStdio,
       });
 
-      testUsingContext('AnsiSpinner works for $testOs', () async {
+      testUsingContext('AnsiSpinner works for $testOs (2)', () async {
```

Comment:
zomg.  Maybe we should just update the test descriptions to be more specific than "AnsiSpinner works..."?

Reply:
can you come up with a better description?

If I was designing test, I wouldn't have a name at all, I'd just identify tests from their filename/line number location, as determined by dredging the stack...